### ISSUE TITLE
Virtual Makeover of a face by facial landmark detection.

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#68]
![image](https://user-images.githubusercontent.com/57909583/103274186-1562c680-49e7-11eb-8be9-e42522724692.png)

We have seen in instagram and snapchat the virtual makeover.

The detected landmarks of faces are used to the calculate contours of the mouth, eyes etc. to add makeup shades virtually.

**Comment by Aayush-hub** [comment]
@akshitagupta15june Mam, I want to work on this, can you please assign me this issue?

**Comment by mehabhalodiya** [comment]
We can add different type of virtual makeover of a face other than #99 ?

**Comment by akshitagupta15june** [comment]
yes you can add

**Comment by akshitagupta15june** [comment]
assigning you

**Comment by rashmitha520** [comment]
Hi I am a gssoc21 participant .Can I work on this issue?

**Comment by Hamza3226** [comment]
Hello, I am a GSSOC'21 participant and would like to contribute to this project.

**Comment by akshitagupta15june** [comment]
yes anyone can contribute to this issue, This issue is open for everyone.

**Comment by manisha-27** [comment]
Hey @akshitagupta15june , please open this issue.

**Comment by akshitagupta15june** [comment]
Reopened

**Comment by ras-29** [comment]
Hello @akshitagupta15june mam, I would like to add "changing eye lens color" under virtual makeover. Please assign this to me.

**Comment by 1anshu-56** [comment]
Hey, I am gssoc participant can you please assign this issue to me as I want to work on this issue.

**Comment by akshitagupta15june** [comment]
ok assigning

**Comment by akshitagupta15june** [comment]
add your implementation here https://github.com/akshitagupta15june/Face-X/tree/master/Virtual_makeover

**Comment by 1anshu-56** [comment]
thanks @akshitagupta15june  for assigning

**Comment by Sash002** [comment]
Can I work on this?

**Comment by akshitagupta15june** [comment]
Yes you can work on it
